_pages/general/visas.md

```diff
@@ -5,17 +5,89 @@ description: How to obtain a visa, useful resources and FAQ's relating to visas
 
 # Visas
 
-If you require a working visa, you'll need to ensure you satisfy the requirements set out by the Australian immigration
-department before you commence our recruitment process. Click [here](https://www.homeaffairs.gov.au/trav/visa-1/482-){:target="\_blank"} to find the most current information relating to the
-482 visa.
+If you require a working visa, you'll need to ensure you satisfy the requirements set out by the Australian immigration department before you commence a recruitment process. Click [here](https://www.homeaffairs.gov.au/trav/visa-1/482-){:target="\_blank"} to find the most current information relating to the 482 visa.
 Once your visa is approved, you can get started setting up things you'll need for life in Australia. For more
-information on 482 visas, please visit the FAQs section.
+information on 482 visas.
 
-## How long does the application process take?
+## FAQ's
+
+### What type of visa should I apply for?
+
+**Skilled Independent (Subclass 189)**
+
+If you are a qualified professional or trades person with work experience that will benefit the Australian economy, and your occupation is on Medium and Long-term Strategic Skills List (MLTSSL), this visa is for you.
+
+In addition to meeting the basic requirements mentioned above, you are also required to pass a points test with points being awarded for age, English language ability, specific work experience, qualification, spouse skills, Australian skilled work experience, Australian qualifications and fluency in one of Australia's major community languages.
+
+The current passmark for this particular visa category is 60 points however, because of ceilings imposed on certain occupations, you may in fact be required to obtain more than 60 points in order to receive an invitation to apply for this visa.
+
+To be able to apply for this visa you need to submit an expression of interest and then be invited through SkillSelect to apply.
+
+This is a permanent residence visa. It lets you and any member of your family unit who has also been granted this visa:
+
+- stay in Australia indefinitely
+- work and study in Australia
+- enrol in Medicare, Australia's scheme for health-related care and expenses
+- apply for Australian citizenship (if you are eligible)
+- sponsor eligible relatives for permanent residence
+- travel to and from Australia for five years from the date the visa is granted (after that time, you will need a - resident return visa or another visa to return to Australia).
+
+**Skilled Nominated (Subclass 190)**
+
+If you are a qualified professional or trades person with work experience that will benefit the Australian economy, and your occupation is NOT on the Medium and Long-term Strategic Skills List (MLTSSL), but is on the Short-term Skilled Occupation List (STSOL) as well as being on one of the State/Territory Migration Plans (STMPs), then you may be eligible for this visa.
```

Comment:
worth adding links?

Reply:
added in 7ea35071100cc60f45aeb4c4fd1a3dc44590f012